src/content/docs/tips/faq.md

```diff
@@ -0,0 +1,52 @@
+---
+title: FAQ
+---
+
+Welcome to the Bitaxes FAQ section. Click on a question below to expand and read the answer.
+
+<details>
+<summary>My Bitaxe isn't hashing at the expected rate</summary>
+<p>
+  If you're experiencing reduced hash rates, try these troubleshooting steps:<br>
+  1. First, physically disconnect and reconnect the power (don't just restart through the UI)<br>
+  2. Check for overheating issues - ensure you're running firmware v2.3 or later to see temperature warnings<br>
+  3. Verify your frequency and voltage settings in the UI under settings - reset to defaults if necessary<br>
+  4. Confirm the input voltage is between 4.8-5.3v<br>
+  5. If overclocking, you may need to increase voltage to match your frequency settings<br>
+  6. As a last resort, try reflashing the firmware
+</p>
+</details>
+
+<details>
+<summary>What should I know about overclocking?</summary>
+<p>
+  Warning: Overclocking can permanently damage your device. If you choose to overclock:<br>
+  • Monitor dashboard values carefully, staying within recommended ranges listed below<br>
+  • Account for ambient temperature fluctuations by maintaining a safety margin<br>
+  • Ensure your power supply is adequate - voltage dropping below 4.8v indicates insufficient power. A higher-rated power supply may be necessary to support increased power demands<br>
+  • For best results, upgrade your heatsink and thermal paste before attempting overclocking
+</p>
+</details>
+
+<details>
+<summary>What are the safe operating ranges?</summary>
+<p>
+Input Voltage: 4.8-5.3v<br>
+ASIC Temperature: Below 70°C<br>
+Voltage Regulator: Below 90°C<br>
+</p>
+</details>
+
+<details>
+<summary>Can I use the same update on my Bitaxe, Hex, and TinyChip boards?</summary>
+<p>
+  No, each have their own repositories and updates.
+</p>
+</details>
+
+<details>
+<summary>Where can I find more information or support?</summary>
+<p>
+  Visit the discord: https://discord.gg/bitaxes
```

Comment:
https://discord.gg/osmu

Reply:
Oops!  Thanks, was a placeholder link until I looked up the correct one.  Updated now